tool/README.md

```diff
@@ -43,15 +47,13 @@ Verify the version changes:
 
 These packages always have their version numbers updated in lock, so we don't have to worry about versioning.
 
-#### Manually review the CHANGELOG.md
-> For both the `$DEVTOOLS_RELEASE_BRANCH` and the `$DEVTOOLS_NEXT_BRANCH` branches
+#### Manually review the CHANGELOG.md in `$DEVTOOLS_RELEASE_BRANCH`
 
-* Verify
-   * that the version for the CHANGELOG entry was correctly generated
-   * that the entries don't have any syntax errors.
+Verify:
+   * the version for the CHANGELOG entry was correctly generated
```

Comment:
can you add a note here to confirm that this set of changelog entries contains all commits from the last minor release. Sometimes if there was a cherry pick release, the generate_changelog script gets confused and will not include commits before the date of the cherrypick

Reply:
can I do it in next PR later today?

Reply:
did we have cherry pick for this release?